lib/api-wmata.js

```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+
+var props = require('../resources/properties');
+var secrets = require('../secret/secrets');
+
+var wmataReq = request.defaults({
+  baseUrl: props.wmata_base_url,
+  qs: secrets.apiKey
+});
+
+var WmataApi = function() {};
+
+WmataApi.prototype.get = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      console.error(endpoint.concat(': Error with WMATA'), error);
+      response.tell(props.wmataErrorSpeechOutput);
+    }
+  });
+};
+
+module.exports = new WmataApi();
```

Comment:
I like separating this out, but it's not necessary to create a `WmataApi` object that doesn't really do anything. We could just export the `get` method.